app/controller/toolbar/CircleSelectionTool.js

```diff
@@ -0,0 +1,90 @@
+/**
+ * This class is the controller for the CircleSelectionController.
+ */
+Ext.define('CpsiMapview.controller.toolbar.CircleSelectionController', {
```

Comment:
The class name should reflect the the JS file incl. its path:

Due to the path of the source file the class should be named the following: `CpsiMapview.controller.toolbar.CircleSelectionTool`

Reply:
solved in https://github.com/compassinformatics/cpsi-mapview/pull/177/commits/b79cde00e7b4d84cc737d59866c7570cad768ee5